pyiron_lammps/compatibility/file.py

```
@@ -0,0 +1,155 @@
+        lmp_str_lst += ["run {} ".format(n_ionic_steps)]
+    elif calc_mode == "minimize":
+        lmp_str_lst += calc_minimize(**calc_kwargs)
+    else:
+        raise ValueError(
+            f"calc_mode must be one of: static, md or minimize, not {calc_mode}"
+        )
+
+    with open(os.path.join(working_directory, "lmp.in"), "w") as f:
+        f.writelines([l + "\n" for l in lmp_str_lst])
+
+    shell = subprocess.check_output(
```

Ruff (0.14.5)

`subprocess` call with `shell=True` identified, security issue

(S602)